pages/roadmap.js

```diff
@@ -0,0 +1,76 @@
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Nobody does only event-driven architectures. Most people complement them with REST (OpenAPI), GraphQL, and/or gRPC APIs. <strong>We want people to use the AsyncAPI specification and tooling together with their existing OpenAPI, GraphQL, and gRPC definitions.</strong> For that purpose, our specification and tools will need to understand and leverage many other specifications and tools.
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    This is not about reinventing the wheel or creating yet another spec to solve the same problems but to integrate with the existing tools instead.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    API development experience is seamless
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    We want to make the development experience super seamless, from idea to production. <strong>Users should be able to create their first API within minutes without prior knowledge of AsyncAPI, the communication protocol, or anything else that's not business logic.</strong>
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    Furthermore, making sure production behavior and documentation are always aligned, independently of the programming language or framework of choice.
+                  </p>
+                </div>
+                <div>
+                  <h2 className="font-semibold mt-6 mb-2 lg:my-0 lg:text-center lg:text-2xl">
+                    <div className="uppercase inline-block bg-gray-100 text-green-400 text-xs p-1.5 mr-2 lg:block lg:bg-transparent lg:text-green-400 lg:text-sm lg:p-0 lg:mr-0 lg:mb-2">Goal</div>
+                    The AsyncAPI community grows 400%
+                  </h2>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    We are aware that our goals are insane. <strong>None of the other goals are possible if we don't have a huge community supporting us. AsyncAPI is and must continue being a community-driven initiative. Now more than ever.</strong>
+                  </p>
+                  <p className="my-4 text-gray-600 lg:text-center">
+                    By 2023, our community has to multiply its size by four when compared with 2021. We need people to contribute code, ideas, docs, articles, presentations, and more. The sponsors base should also grow along with the community size and the donated money should serve to give back to the community.
```

Comment:
Mentioning **2023** sounds contradictory (at the same time sounds like a constraint) since the vision specifies 2026.

Reply:
The vision is 5 years, this specific goal is for 2 years.

Reply:
I can see it's misleading. I'll just remove the time constraint in this paragraph.